test/storage/sql_table_test.cpp

```diff
@@ -536,7 +536,7 @@ TEST_F(SqlTableTests, ScanTest) {
   table.table_->Scan(txn, &start_pos, pc, pc_pair.second, table.version_);
 
   // check the number of tuples we found
-  EXPECT_EQ(pc->NumTuples(), 4);
+  EXPECT_EQ(pc->NumTuples(), 2);
```

Comment:
I am not following this. Are we changing the semantics of Scan? We inserted 4 tuples but why we should only get 2?